nixos/configuration.nix

```diff
@@ -4,13 +4,11 @@
   imports = [ 
     ./hardware-configuration.nix 
     ./guest.nix
-    ./users.nix
-    ./vagrant.nix
   ];
 
+  users.mutableUsers = false;
   # we always want git and vim
   environment.systemPackages = with pkgs; [ 
```

Comment:
Remove both git and vim. We don't need to promote artisinal hand crafted edits on images produced. :)